content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -128,3 +127,40 @@ spec:
       port: 9080
       weight: 10
 {{< /text >}}
+
+## Unsupported APIs with waypoints
+
+Some legacy Istio APIs are deliberately not supported by waypoints in ambient mode.  These APIs can still be used with [Istio Gateways](/docs/tasks/traffic-management/ingress/ingress-control/).
```

Comment:
Should we call out that these APIs can be used with Istio Gateways even if they're instantiated with Gateway API?

Reply:
that's exactly the problem - if we insert the word "Classic" to this sentence it becomes clear that Istio Gateways cannot be instantiated with Gateway API :)